### DISCUSSION TITLE
✅ 🚀 Feature Request: Provide a way to disable the generator-tag

### DISCUSSION THREAD

**Discussion by tobias992** [elementor/elementor#22561] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Elementor bloats up the header with generator-tag. Why? In my case i found this in source code:

`<meta name="generator" content="Elementor 3.13.3; features: e_dom_optimization, e_optimized_assets_loading, e_optimized_css_loading, e_font_icon_svg, a11y_improvements; settings: css_print_method-external, google_font-enabled, font_display-auto">`

### Describe the solution you'd like

Remove it. Nobody needs.

### Describe alternatives you've considered

Add option in settings to remove manually.

**Comment by nicholaszein** [comment]
Hello there @tobias992!

This option is already available in your WordPress `Dashbaord` > `Elementor` > `Settings` > `Advanced` page.

![image](https://github.com/elementor/elementor/assets/113993460/4742bb6f-ffec-4a83-9a83-f709b7b08064)


Kind regards
